### PR TITLE
Valentine Cards

### PR DESCRIPTION
**Pull request by Maleclypse**
#### Summary
Content "Valentine Day Cards"

#### Purpose of change

Add Valentine day cards.

#### Describe the solution
Add's the item and snippets.  Need to add item spawns. 

#### Describe alternatives you've considered

None

#### Testing
TBD

### CODE REVIEW

File: data/json/items/newspaper.json [hunk 1]
```diff
@@ -14,6 +14,21 @@
...
   },
+  {
+    "type": "GENERIC",
+    "id": "valentine_card",
+    "category": "books",
+    "symbol": ",",
+    "color": "red",
+    "name": { "str": "valentine card" },
+    "snippet_category": "valentine",
+    "description": "A creased holiday card.  It appears to be a valentine's day card from the last year.",
```
Comment (actual-nh) [inline data/json/items/newspaper.json:25]:
```suggestion
    "description": "A creased holiday card.  It appears to be a Valentine's Day card from the previous year.",
```

File: data/json/snippets/valentines.json [hunk 2]
```diff
@@ -0,0 +1,36 @@
+[
+  {
+    "type": "snippet",
+    "category": "valentine",
+    "text": [
+      { "id": "valentine_1", "text": "There is a picture of a kitten pouncing, blood faintly drips from it's claws.  \"I'd kill!  For you to be my valentine.\"" },
```
Comment (actual-nh) [inline data/json/snippets/valentines.json:6]:
```suggestion
      { "id": "valentine_1", "text": "There is a picture of a kitten pouncing; blood faintly drips from its claws.  \"I'd kill!  For you to be my valentine.\"" },
```

File: data/json/snippets/valentines.json [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+      { "id": "valentine_5", "text": "A girl with a giant spoon stirs a cauldron with a boy inside and a fire below.  \"You suit my taste just right, Valentine\"" },
+      {
+        "id": "valentine_6",
+        "text": "A person stepping out from the forest.  \"Don't be afraid, it's me, your valentine!\""
+      },
+      {
+        "id": "valentine_7",
+        "text": "A building that says 'Crematorium' on it.  \"Ashes to ashes, dust to dust, show me a Valentine, that I can trust\""
+      },
+      { "id": "valentine_8", "text": "A popular RPG character.  \"We should adventure together.\"  A village in flames." },
```
Comment (actual-nh) [inline data/json/snippets/valentines.json:25]:
```suggestion
      { "id": "valentine_8", "text": "A popular RPG character.  \"We should adventure together.\"  In the background, a village in flames." },
```

File: data/json/snippets/valentines.json [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+      {
+        "id": "valentine_9",
+        "text": "A picture of a room that looks like your bedroom.  \"I can't wait to see you again, Valentine.  Sweet dreams.\""
+      },
+      {
+        "id": "valentine_10",
+        "text": "A girl smiles into the camera.  Behind here a house is on fire.  \"I'm burning, I'm burning, I'm burning for you, valentine\""
```
Comment (actual-nh) [inline data/json/snippets/valentines.json:32]:
```suggestion
        "text": "A girl smiles into the camera.  Behind here a house is on fire.  \"I'm burning, I'm burning, I'm burning for you, my valentine!\""
```

**Review by actual-nh — COMMENTED** [review]
Amusing!

**Comment by topherclay** [comment]
These are so good! :D

**Comment by actual-nh** [comment]
@KorGgenT: How about a Kelvinist version of the 10th one, with the girl looking through a flame on her palm (or something like that)?

File: data/json/items/newspaper.json [hunk 5]
```diff
@@ -14,6 +14,21 @@
...
   },
+  {
+    "type": "GENERIC",
+    "id": "valentine_card",
+    "category": "books",
+    "symbol": ",",
+    "color": "red",
+    "name": { "str": "valentine card" },
+    "snippet_category": "valentine",
+    "description": "A creased holiday card.  It appears to be a Valentine's Day card from the previous year.",
```
Comment (Xaleth) [inline data/json/items/newspaper.json:25]:
```suggestion
    "description": "A creased holiday card.  It appears to be a Valentine's Day card from before the Cataclysm.",
```

**Comment by UmbralReaper** [comment]
I see the Cataclysm also effected people's taste in cards.

**Pull request merged**